Etabs_Adapter/AdapterActions/Exectute.cs

```diff
@@ -0,0 +1,174 @@
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System.Collections.Generic;
+using System.Collections;
+using BH.oM.Adapter;
+using BH.oM.Reflection;
+using BH.oM.Adapter.Commands;
+using BH.oM.Structure.Loads;
+
+#if Debug17 || Release17
+using ETABSv17;
+#else
+using ETABS2016;
+#endif
+
+
+namespace BH.Adapter.ETABS
+{
+#if Debug17 || Release17
+    public partial class ETABS17Adapter
+#else
+    public partial class ETABS2016Adapter
+#endif
+    {
+        /***************************************************/
+        /**** IAdapter Interface                        ****/
+        /***************************************************/
+
+        public override Output<List<object>, bool> Execute(IExecuteCommand command, ActionConfig actionConfig = null)
+        {
+            var output = new Output<List<object>, bool>() { Item1 = null, Item2 = false };
+
+            output.Item2 = RunCommand(command as dynamic);
+
+            return output;
+        }
+
+        /***************************************************/
+        /**** Commands                                  ****/
+        /***************************************************/
+
+        public bool RunCommand(NewModel command)
+        {
+            bool success = m_model.InitializeNewModel(eUnits.N_m_C) == 0;
+            success &= m_model.File.NewBlank() == 0;
+            return success;
+        }
+
+        /***************************************************/
+
+        public bool RunCommand(Save command)
+        {
+            return m_model.File.Save() == 0;
+        }
+
+        /***************************************************/
+
+        public bool RunCommand(SaveAs command)
+        {
+            return m_model.File.Save("@" + command.FileName) == 0;
```

Comment:
The `"@"` makes it so that the default grasshopper path components output does not work.
Example: `C:\Users\kthorsager\OneDrive - BuroHappold\Documents\ETABS buggs\ETABSSaveAsTest.edb`
When removed it does.

Reply:
thanks, fixed.